lib/util/http/index.ts

```diff
@@ -172,6 +172,23 @@ export class Http<GetOptions = HttpOptions, PostOptions = HttpPostOptions> {
     return this.request<string>(url, { ...options, method: 'head' });
   }
 
+  protected requestBuffer<T>(
+    url: string | URL,
+    httpOptions?: InternalHttpOptions
+  ): Promise<HttpResponse<T> | null> {
+    return this.request<T>(url, {
+      ...httpOptions,
+      responseType: 'buffer' as any,
+    });
+  }
+
+  getBuffer(
+    url: string,
+    options: HttpOptions = {}
+  ): Promise<HttpResponse<Buffer> | null> {
+    return this.requestBuffer(url, options);
+  }
```

Comment:
Why two functions?

Is `Buffer` supported by `got`

Reply:
> Why two functions?

Aligned with how the existing methods are organized.

> Is `Buffer` supported by `got`

Yes. https://github.com/sindresorhus/got/blob/v11.8.2/source/as-promise/parse-body.ts#L20-L22